### DISCUSSION TITLE
Is there any sort of migration guide for going from scikit-build to scikit-build-core?

### DISCUSSION THREAD

**Discussion by vyasr** [scikit-build/scikit-build#875] [Q&A]
I would be happy to contribute such a guide as part of the process of making that transition if one doesn't exist. In that case I would probably be posting any relevant questions back to this discussions board.

**Comment by henryiii** [comment]
Not yet, I'd be happy to have you work on that if you'd like to. I'll have tutorial material soon, and once we have edible builds, I'll start putting together documentation for scikit-build-core (possibly merging the docs into scikit-build.org).

**Comment by vyasr** [comment]
Mm tasty edible builds 

Sure, I'll probably approach this by opening a draft PR where I keep a running list of items (including tasks that cannot be completed based on the current state of scikit-build-core) and once any project that I maintain is able to fully transition to it I can open that PR up for merge and we can make further improvements incrementally.
